components/proxy/conf/Caddyfile

```diff
@@ -174,6 +173,10 @@ api.{$GITPOD_DOMAIN} {
 		allowed_origins https://{$GITPOD_DOMAIN}
 	}
 
+	handle /gitpod.v1.* {
```

Comment:
FYI, when public services are enabled on dedicated, only `/api` and `/idp` are exposed. That means this is not going to be available in those networking modes

Reply:
I do not know if `api.` is exposed with HTTP 2, you'll need to test yourself

Reply:
> That means this is not going to be available in those networking modes to the public internet

Ok, but I guess that is kind of expected.

> I do not know if api. is exposed with HTTP 2, you'll need to test yourself

:+1: Will test on dogfood then. :eyes: